frontend/src/pages/LoginPage.tsx

```diff
@@ -95,6 +95,7 @@ export default function LoginPage() {
       dispatch(setCredentials({ token, user }));
       navigate('/dashboard/Approve Forms');
     } catch (unparsdError) {
+      // FIXME: api error response types please
```

Comment:
error types start to harden the API. if we're doing { status: string, data: string } format, we should be quite talkative before going to custom error typing over passing up verbose error from where the error was created.